app/views/credits/index.html.erb

```diff
@@ -2,7 +2,6 @@
   <h1>Credits</h1>
 
   <div class="content">
-    <%= managed_content("credits") %>
+    <% html(@page.body) %>
```

Comment:
I think this was working before #440 was merged, but now we need an equals here (ie `<%= html(@page.body) %>`).